legacy/src/main/java/org/opensearch/sql/legacy/pit/PointInTimeHandler.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.legacy.pit;
+
+import lombok.Getter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.search.CreatePitRequest;
+import org.opensearch.action.search.CreatePitResponse;
+import org.opensearch.action.search.DeletePitRequest;
+import org.opensearch.client.Client;
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.core.action.ActionListener;
+
+public class PointInTimeHandler {
+  private Client client;
+  @Getter private String pitId;
+  private static final Logger LOG = LogManager.getLogger();
+
+  public PointInTimeHandler(Client client, String[] indices) {
+    this.client = client;
+
+    CreatePitRequest createPitRequest = new CreatePitRequest(new TimeValue(600000), false, indices);
```

Comment:
Should we use `plugins.sql.cursor.keep_alive` instead of `6000000` here?

Reply:
yes. thanks for catching this.